src/integration-test/java/com/commercetools/sync/integration/commons/utils/ITUtils.java

```diff
@@ -323,12 +325,30 @@ private static AssetDraftBuilder createAssetDraftBuilder(@Nonnull final String a
      *         the supplied {@code assetDrafts}.
      */
     public static ProductVariantDraft createVariantDraft(@Nonnull final String variantKeyAndSku,
-                                                         @Nonnull final List<AssetDraft> assetDrafts) {
+                                                         @Nullable final List<AssetDraft> assetDrafts) {
+
```

Comment:
@jortizsao 
- It was more meant as sugar, instead of having to pass always `createVariantDraft(key, assets, null)` to just pass `createVariantDraft(key, assets)`. 
- Yes, we could also have more sugars like `createVariantDraft(key, prices)` and `createVariantDraft(key)`

It's now a matter of having several methods for each combination VS passing a lot of `nulls` with one method.

Advantages

1. Better looking method calls (no nulls passing) which makes tests easier to the eye. 

Disadvantages

1. Method caller doesn't explicitly see that the parameters are `null`.
2. More methods at the utils for each combination.

In that case I see more disadvantages so I would stick to having one method that could take `null`s (your opinion). What do you think?

Reply:
I agree that for this case having one method is better (less is more :) )